swarm/src/connection/pool.rs

```diff
@@ -513,11 +512,13 @@ where
             waker.wake();
         }
 
-        let connection = Connection::new(
+        let span = tracing::error_span!("connection", connection_id = %id, peer_id = %obtained_peer_id, peer_address = ?endpoint);
```

Comment:
```suggestion
        let span = tracing::error_span!("connection", id = %id, peer = %obtained_peer_id, peer_address = ?endpoint);
```

Let's shorten this, otherwise it is super noisy.